composer.json

```diff
@@ -1,7 +1,7 @@
 {
 	"name": "coloredcow/laravel-audit",
 	"description": "Audit logging for Laravel",
-	"version": "2.1.0",
+	"version": "3.0.0",
```

Comment:
Can we change version as `3.0` is already taken on laravel-9 support?

Reply:
@Shailesh-Kala I have changed the version please have a look.